src/content/pages/FileUpload/ParameterForm.tsx

```diff
@@ -0,0 +1,122 @@
+﻿import React, { useEffect, useState, useRef } from 'react'
+import {
+  Box,
+  FormControlLabel,
+  IconButton,
+  Modal,
+  Switch,
+  Tab,
+  Tabs,
+  Typography,
+  TableContainer,
+  Table,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell
+} from '@mui/material'
+import HelpIcon from '@mui/icons-material/Help'
+import { modalStyle } from './modalStyle'
+import CancelIcon from '@mui/icons-material/Cancel'
+import ParameterModal from './ParameterModal'
+
+import parameters from '../../../../../sharedData/parameterBuckets.json'
+import { NullableBooleanInput } from 'react-admin'
+
+const intrinsicParameters = parameters.intrinsicParameters
+const extrinsicParameters = parameters.extrinsicParameters
+const otherParameters = parameters.otherParameters
+
+
+
+function ParameterForm({ selectedBuckets, setSelectedBuckets }) {
+  const [intrinsic, setIntrinsic] = useState(false)
+  const [extrinsic, setExtrinsic] = useState(false)
+  const [other, setOther] = useState(false)
+  const [all, setAll] = useState(false)
+  const [openParameterModal, setOpenParameterModal] = useState(false)
+
+  const parameters = { intrinsic: intrinsicParameters, extrinsic: extrinsicParameters, other: otherParameters }
```

Comment:
```suggestion
function ParameterForm({ setSelectedBuckets }) {
  const [intrinsic, setIntrinsic] = useState(false)
  const [extrinsic, setExtrinsic] = useState(false)
  const [other, setOther] = useState(false)
  const [all, setAll] = useState(false)
  const [openParameterModal, setOpenParameterModal] = useState(false)
```

`intrinsicParameters`, `extrinsicParameters`, `otherParameters`, and the `parameters` object defined using them are currently unused, and should be removed.

Similarly, the `selectedBuckets` parameter of `ParameterForm` is also unused, and should be removed (`FileUpload/index.tsx` line 94 should also be changed to reflect this)

Reply:
Adressed this